tests/src/test/scala/com/nvidia/spark/rapids/OrcScanSuite.scala

```diff
@@ -105,10 +105,9 @@ class OrcScanSuite extends SparkQueryCompareTestSuite {
    * then no result returned. Because of 1582-10-03 in hybrid calender
    * is actually 1582-09-23 in proleptic Gregorian calendar.
    */
-  test("test hybrid Julian Gregorian calendar vs proleptic Gregorian calendar") {
+  ignore("test hybrid Julian Gregorian calendar vs proleptic Gregorian calendar") {
```

Comment:
Because it will fail due to a bug in ORC, previous code had an assumption that it would only run on Spark prior to 3.1.1.  If we'd rather just remove the test, I can do that instead.

Reply:
Yes, the issue is https://issues.apache.org/jira/browse/ORC-1083 that we filed.  When that gets fixed, we could turn this test back on for Spark versions that pick up that change.  If you feel that's not worth keeping the test around for, I'll go ahead and remove it.

Reply:
Never mind I see it is there already